core/templates/dev/head/pages/story-viewer-page/chapters-list/story-viewer-chapters-list.directive.html

```diff
@@ -1,7 +1,15 @@
 <div>
+  <background-banner></background-banner>
   <div ng-if="$ctrl.storyPlaythroughObject" class="oppia-story-player-tiles-container">
-    <div class="oppia-story-path-section d-none d-md-block">
-      <svg width="500px" ng-attr-height="<[$ctrl.svgHeight + 'px']>" xmlns="http://www.w3.org/2000/svg">
+    <div class="oppia-page-heading oppia-story-title">
+      <[$ctrl.getTitle()]>
+    </div>
+    <div class="oppia-story-description">
+      <[$ctrl.getDescription()]>
+    </div>
+
+    <div ng-if="$ctrl.storyPlaythroughObject" class="oppia-story-path-section d-none d-md-block">
```

Comment:
Slightly confused about the ng-if here. Don't you already have that check in line 3?

Reply:
Oh, yeah, will remove it.

Reply:
Done